doc/guide.rst

```diff
@@ -750,3 +750,53 @@ an input to the above mentioned genotyping commands. This can be done by running
     whatshap find_snv_candidates ref.fasta input.bam -o variants.vcf
 
 If Nanopore reads are used for calling SNPs, it is recommended to add option --nanopore to the above command.
+
+Polyploid Phasing
+=================
+
+In addition to diploid phasing, WhatsHap also supports polyploid phasing 
+through a different algorithm. The ``whatshap polyphase`` command works 
+almost the same as the ``phase`` command with a few restrictions:
+
+1. An additional integer argument ``--ploidy`` must be specified. This ploidy
+must match the ploidy in the provided VCF file(s). The ploidy also greatly
+impacts the running time as the phasing becomes more complex. Ploidies
+higher than 6 may take very long to process.
+
+2. WhatsHap will use available genotype information from the VCF file(s), but
+the computed haplotypes are not guaranteed to follow these genotypes, if they
+deviate too much from the allele distribution among the aligned reads.
+
+3. Polyploid phasing on pedigrees is not supported yet.
+
+4. Structural variants are not considered during the phasing. The algorithm
+will always produce the provided number of haplotypes at any location.
+
+Opposed to the diploid phasing algorithm, there is no strict limitation
+regarding the coverage of the input reads. However, it still increases the
```

Comment:
I think this statement on the diploid algorithm is confusing and would remove it.